guide/getting-started/getting-started.md

```diff
@@ -34,70 +34,68 @@ Illustration sources
-
 ---
 
 ### [Why Bitcoin is unique]({{ '/guide/getting-started/why-bitcoin-is-unique/' | relative_url }})
 
-The internet has opened an infinite design space for information and communication. Bitcoin adds the new dimension of transferring value that could bring similar change as previous waves of innovation.
+Familiarize yourself with some of the most significant concepts that bitcoin designers need to know.
 
 ---
 
 ### [Why design for Bitcoin]({{ '/guide/getting-started/why-design-for-bitcoin/' | relative_url }})
 
-Bitcoin is fundamentally open and participatory and allows designers and creators to explore and shape experiences for users around the globe.
+Bitcoin is open-source, decentralized, permissionless, fungible, stateless, and non-custodial. If those words don’t mean anything to you, consider starting here.  
 
 ---
 
 ### [Open design]({{ '/guide/getting-started/open-design/' | relative_url }})
 
-Open design is about global collaboration that balances the free exchange of design information, the independence of individual designers and the collective power of collaboration.
+Bitcoin isn’t just changing money—it’s changing how designers collaborate across broad use cases. While open-source development has been around for years, open design is brand new.
 
 ---
 
 ### [The visual language of Bitcoin]({{ '/guide/getting-started/visual-language/' | relative_url }})
 
-As an open and public protocol, Bitcoin does not have a strictly defined brand. Yet there is a recognizable symbol that has evolved and remixed endlessly.
+Since bitcoin is public, open, and dependent upon no single person or organization, its brand is still undefined—and it might always be. But there’s still a to learn about how best to represent it visually.
 
 ---
 
 ### [User research]({{ '/guide/getting-started/user-research/' | relative_url }})
 
-All products begin with user needs and end in those same users hands. User research is a fundamental activity that helps us focus on things people care about.
+Before a product can solve its users’ problems, its designers need to understand what those problems are. User research helps us zoom in on what users care about.
 
 ---
 
 ### [Personal finance]({{ '/guide/getting-started/personal-finance/' | relative_url }})
 
-If we consider Bitcoin money, we need to consider how Bitcoin users think about their finances. This page provides a simple framework that can serve as a starting point, deriving bitcoin software and hardware setups from common budgeting categories.
+The language of bitcoin is the language of money. If you’re still new to the concepts that underpin personal finance, this section can get you up to speed.
```

Comment:
Also misses the point a little. It's not just about personal finance, but how Bitcoin products can be used to address those different personal finance needs.

Reply:
Need a concrete suggestion here.

Reply:
Just keep the original text?